integrations/mongodb_atlas/pyproject.toml

```diff
@@ -0,0 +1,182 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "mongodb-atlas-haystack"
+dynamic = ["version"]
+description = "An integration of MongoDB Atlas with Haystack"
+readme = "README.md"
+requires-python = ">=3.8"
+license = "Apache-2.0"
+keywords = []
+authors = [
+  { name = "deepset GmbH", email = "info@deepset.ai" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = [
+  "haystack-ai>=2.0.0b6",
+  "pymongo[srv]",
+]
+
+[project.urls]
+Source = "https://github.com/deepset-ai/haystack-core-integrations"
+Documentation = "https://github.com/deepset-ai/haystack-core-integrations/blob/main/integrations/mongodb_atlas/README.md"
+Issues = "https://github.com/deepset-ai/haystack-core-integrations/issues"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/haystack_integrations"]
+
+[tool.hatch.version]
+source = "vcs"
+tag-pattern = 'integrations\/pgmongodb-atlasvector-v(?P<version>.*)'
```

Comment:
This tag is wrong

Reply:
This was fixed right after the merge: https://github.com/deepset-ai/haystack-core-integrations/pull/426